lib/elixir/lib/regex.ex

```diff
@@ -348,6 +348,9 @@ defmodule Regex do
       order. Defaults to `:first` which means captures inside the regex do not
       affect the splitting process.
 
+    * `:include_captures` - when `true`, includes in the result the matches of
```

Comment:
dunno, using only `:captures` feels not clear enough to me, if I have to think about reading code wrote by someone else. es: `Regex.split(~r/abc/, "abcdefg", captures: true)`. Who captures what? I think that removing the prefix is not the right choice.